prac_06/guitars.py

```diff
@@ -0,0 +1,29 @@
+from prac_06.guitar import Guitar
+
+
+def main():
+    """Get information about users guitars and print information."""
+    guitars = []
+    # guitars.append(Guitar("Gibson L-5 CES", 1922, 16035.40))
+    # guitars.append(Guitar("Line 6 JTV-59", 2010, 1512.9))
+
+    print("My guitars!")
+    name = input("Name: ")
+    while name != "":
+        year = int(input("Year: "))
+        cost = float(input("Cost: "))
+        guitars.append(Guitar(name, year, cost))
+        print()
```

Comment:
After you store the guitar information you should include a line to tell the user e.g. 
"""
My guitars!
Name: Fender Stratocaster
Year: 2014
Cost: $765.4
Fender Stratocaster (2014) : $765.40 added.
"""

Reply:
Good catch, thank you.